angular/templates/profile-index.html

```diff
@@ -32,8 +32,8 @@
 						<h4>{{item.artist}}</h4>
 						<h4>{{item.album}}</h4>
 					</div>
-				</div uib-slide>
-			</div uib-carousel>
```

Comment:
@adamlucz90 You don't need to add the boostrap attributes to the close tags, only the opening tags.